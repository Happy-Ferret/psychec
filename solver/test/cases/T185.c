void f()
{
    char  *err;
    parse_user_spec = err;

    ignore_value (parse_user_spec);
    ignore_value (parse_additional_groups);

    if (parse_additional_groups != 0)
    {
    }
}

void f2()
{
    char const *err2;
    err2= parse_user_spec2;

    ignore_value2 (parse_user_spec2);
    ignore_value2 (parse_additional_groups2);

    if (parse_additional_groups2 != 0)
    {
    }
}

void f3()
{
    char const *err3 = parse_user_spec3;

    ignore_value3 (parse_user_spec3);
    ignore_value3 (parse_additional_groups3);

    if (parse_additional_groups3 != 0)
    {
    }
}

void f4()
{

    ignore_value4 (parse_user_spec4);
    ignore_value4 (parse_additional_groups4);

    char const *err4 = parse_user_spec4;

    if (parse_additional_groups4 != 0)
    {
    }
}
