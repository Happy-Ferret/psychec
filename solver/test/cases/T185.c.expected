/* Forward declarations */

/* Type definitions */

/* Variables and functions */
 int /*<<< orphan*/  ignore_value (char*) ; 
 int /*<<< orphan*/  ignore_value2 (char*) ; 
 int /*<<< orphan*/  ignore_value3 (char*) ; 
 int /*<<< orphan*/  ignore_value4 (char*) ; 
 char* parse_additional_groups ; 
 char* parse_additional_groups2 ; 
 char* parse_additional_groups3 ; 
 char* parse_additional_groups4 ; 
 char* parse_user_spec ; 
 char* parse_user_spec2 ; 
 char* parse_user_spec3 ; 
 char* parse_user_spec4 ; 

